docs/pages/desktop-access/reference/configuration.mdx

```diff
@@ -55,14 +55,28 @@ kind: windows_desktop
 metadata:
   name: fixed-screen-size
 spec:
-  host_id: 307e091b-7f6b-42e0-b78d-3362ad10b55d
   addr: 192.168.1.153:3389
   non_ad: true
+  # host_id is the `metadata.name` of the `windows_desktop_service` resource
+  # reporting this desktop.
+  #
+  # For AD-joined hosts, this value MUST be the `name` of the `windows_desktop_service`
+  # resource that connects to the AD domain via LDAP. For hosts that use local accounts
+  # (`non_ad: true`), this value MUST be the `name` of a `windows_desktop_service`
+  # that can access the RDP port of the host (`spec.addr` above).
+  #
+  # `windows_desktop` resources are identified by the combination of `metadata.name` and `spec.host_id`.
```

Comment:
Once again, this is incorrect.

Any w_d_s with network connectivity will do.

For AD, there needs to be a w_d_s configured for the particular domain (to ensure that CRLs are published and NTAuth is up to date), but we don't necessarily need to connect through that w_d_s.

Reply:
Oh, sorry, missed the punchline of that thread.

e52fbcb6b16f5e8073f0f1758b453e31d5ef4855